webapp/src/main/resources/db/migration/V64__third_party_sync_file_checksum.sql

```diff
@@ -0,0 +1,13 @@
+create table third_party_sync_file_checksum (
+    id bigint(20) NOT NULL AUTO_INCREMENT,
+    repository_id bigint not null,
+    locale_id bigint not null,
+    md5 char(32) not null,
+    file_name varchar(255) not null,
+    created_date datetime not null,
+    last_modified_date datetime,
+    primary key (id)
+);
+alter table third_party_sync_file_checksum add constraint FK__THIRD_PARTY_CHECKSUM__REPO__ID foreign key (repository_id) references repository (id);
```

Comment:
similarly to previous comment, the name should include the full table name or a version with initials